src/components/SupportersList.astro

```diff
@@ -0,0 +1,56 @@
+---
+import FormattedDate from '@components/FormattedDate';
+import { getCollection } from 'astro:content';
+
+const supporters = (await getCollection('supporters')).sort(
+	(a, b) => a.data.date.valueOf() - b.data.date.valueOf()
+);
+---
+
+<ul class="supporters">
+  {supporters.map(supporter => (
+    <li>
+      <strong>{supporter.data.name}</strong>
+      
+      { supporter.data.role &&
+        <span>
+          {supporter.data.role} {supporter.data.company && `(${supporter.data.company})`}
+        </span>
+      }
+
+      <span>
+        Signed: {supporter.data.date && <FormattedDate date={supporter.data.date} />}
+      </span>
+    </li>
+  ))}
+</ul>
+
+
+<style>
+  .supporters {
+    margin: 2rem 0 0;
+    padding: 0;
+  }
+
+  .supporters li {
+    list-style-type: none;
+    margin-bottom: 2rem;
+  }
```

Comment:
```suggestion
    margin-top: 0;
  }
```


Could we zero out the top margin so the columns are even? 
![image](https://github.com/email-markup-consortium/emailmarkup.org/assets/10135143/fac86d2f-7643-4977-bfee-dca22be510a7)